util.go

```diff
@@ -82,11 +82,13 @@ func DefaultTableSummary() map[int]func(io.Writer, int) (int, error) {
 //               |                           |
 //
 type LineStyle struct {
-	Top  [4]rune
-	Mid  [4]rune
-	Row  [4]rune
-	Wrap [4]rune
-	End  [4]rune
+	Top          [4]rune
+	Mid          [4]rune
+	Row          [4]rune
+	Wrap         [4]rune
+	End          [4]rune
+	EmbedColumns bool
+	Compact      bool
```

Comment:
I don't believe this option is necessary. This should be the same as borders = 0 and the `Wrap` line's spacer character as `0`.